website/docs/r/vpn_customer_gateway.html.markdown

```diff
@@ -49,6 +49,9 @@ The following arguments are supported:
 * `ike_lifetime` - (Optional) The IKE lifetime of phase 2 VPN connection to this
     VPN Customer Gateway in seconds (defaults 86400)
 
+* `project` - (Optional) The name or ID of the project to deploy this
+    instance to. Changing this forces a new resource to be created.
```

Comment:
I noticed a few resources have this exact description while others have a more specific one. So think that something like `The name or ID of the project to create this VPN Customer Gateway in. Changing this forces a new resource to be created.` would be just a bit niced :wink"